test/e2e/common/configmap.go

```diff
@@ -133,6 +133,26 @@ var _ = Describe("[sig-node] ConfigMap", func() {
 		configMap, err := newConfigMapWithEmptyKey(f)
 		framework.ExpectError(err, "created configMap %q with empty key in namespace %q", configMap.Name, f.Namespace.Name)
 	})
+
+	/*
+	   Release : v1.16
+	   Testname: Update ConfigMap
+	   Description: Attempt to update a ConfigMap after create one.
```

Comment:
Won't block on this, but this isn't RFC2119 language https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/conformance-tests.md#conformance-test-comment-metadata